man/man8/zpool.8

```diff
@@ -2044,6 +2062,17 @@ Because scrubbing and resilvering are \fBI/O\fR-intensive operations, \fBZFS\fR
 .RS 6n
 Stop scrubbing.
 .RE
+.sp
+.ne 2
+.na
+\fB\fB-p\fR\fR
+.ad
+.RS 6n
+Pause scrubbing. Scrub progress is periodically synced to disk so if the system
```

Comment:
I can add a sentence at the end: "Note, resilver can't be paused."
However, this is documenting the scrub command so I'm not sure resilver notes belong here?